### DISCUSSION TITLE
Availability to change Favicons and related

### DISCUSSION THREAD

**Discussion by amauryhubault** [outline/outline#4029] [Ideas]
Hi,

As we're using several instances of Outline (depending on purpose, e.g. wiki is separate from operations follow-up), we'd like to differenciate them on the shortcuts, such as on the iPhone springboard. 
![IMG_E4B291E9CA6F-1](https://user-images.githubusercontent.com/6259850/187421519-6ce384b8-4c59-4c88-9d8e-cbbaca9836d5.jpeg)
Would it be possible to allow admins to change the icons?

**Comment by Limezy** [comment]
If you compile Outline yourself, you can change that icon (I did it)
